transform-plugins/README.rst

```diff
@@ -4,230 +4,213 @@ Hydrator Transform Plugin Collection
 
 Introduction
 ============
-
-This project is a collection of useful transformations of data. Following is list of plugins
-that are currently available:
-
-- CSV Parser,
-- CSV Formatter,
-- JSON Parser,
-- JSON Parser,
-- Clone Record,
-- Stream Formatter,
-- Compressor, 
-- Decompressor,
-- Encoder and
+This project is a collection of useful transformations of data. These plugins are currently available:
+
+- CSV Parser
+- CSV Formatter
+- JSON Parser
+- JSON Parser
+- Clone Record
+- Stream Formatter
+- Compressor
+- Decompressor
+- Encoder
 - Decoder
 
 Getting Started
 ===============
-
-Following are instructions to build and deploy hydrator transform plugins. 
+Follow these instructions to build and deploy Hydrator transform plugins. 
 
 Prerequisites
 -------------
-
-To use plugins, you must have CDAP version 3.2.0 or later. You can download CDAP Standalone that includes Hydrator `here <http://cask.co/downloads>`__
+To use plugins, you must have CDAP version 3.2.0 or later. You can download CDAP Standalone that includes Hydrator `here <http://cask.co/downloads>`__.
   
 Build Plugins
 -------------
-
 You can get started with CDAP-plugins by building directly from the latest source code::
 
   git clone https://github.com/caskdata/hydrator-plugins.git
   cd hydrator-plugins
   mvn clean package -pl transform-plugins
 
-After the build completes, you will have a jar for each plugin under the
+After the build completes, you will have a JAR for each plugin under each
 ``<plugin-name>/target/`` directory.
 
 Deploy Plugins
 --------------
+You can deploy a transform plugin using the CDAP CLI::
 
-You can deploy transform plugins using the CDAP CLI::
-
-  > load artifact target/transform-plugins-1.0-SNAPSHOT-batch.jar \
+  cdap > load artifact target/transform-plugins-1.0-SNAPSHOT-batch.jar \
          config-file resources/plugin/transform-plugins.json
 
-Copy the UI configuration to CDAP installation::
+Copy the UI configuration to the CDAP installation::
 
-  > cp transform-plugins/resources/ui/*.json $CDAP_HOME/ui/templates/common/
+  $ cp transform-plugins/resources/ui/*.json $CDAP_HOME/ui/templates/common/
 
 Plugin Descriptions
 ===================
 
 CSV Parser
 ----------
-
-:Id:
+:ID: 
   **CSVParser**
-:Type:
+:Type: 
   Transform
-:Mode:
-  Batch and
+:Mode: 
+  Batch and 
```

Comment:
I don't think it matters as far as the parser is concerned. However, I can (and will) take them out. Thanks!